documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -156,73 +144,41 @@ claimRef:
-
-. Recreate all `KafkaTopic` resources.
+[source,shell]
+kubectl apply -f <topic_configuration_file>
 +
-It is essential that you recreate the resources before deploying the cluster, or the Topic Operator will delete the topics.
+It is essential that you recreate the resources before deploying the cluster or the Topic Operator will delete the topics.
 
-. Deploy the Kafka cluster.
+. Deploy the Kafka cluster using the original configuration for the `Kafka` resource:
 +
-For example:
-+
-[source,shell,subs="+quotes,attributes"]
+[source,shell]
 ----
-kubectl apply -f _kafka.yaml_
+kubectl apply -f <kafka_resource_configuration>.yaml -n my-project
 ----
---
-+
---
-*_Option 2_*: If you do not have all the `KafkaTopic` resources that existed before you lost your cluster:
-
-. Deploy the Kafka cluster, as with the first option, but without the Topic Operator by removing the `topicOperator` property from the Kafka resource before deploying.
-+
-If you include the Topic Operator in the deployment, the Topic Operator will delete all the topics.
 
-. Delete the internal topic store topics from the Kafka cluster:
+. Verify the recovery by listing the `KafkaTopic` resources:
 +
-[source,shell,subs="+attributes"]
+[source,shell]
 ----
-kubectl run kafka-admin -ti --image={DockerKafkaImageCurrent} --rm=true --restart=Never -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete
+kubectl get kafkatopics -o wide -w -n my-project
 ----
 +
-The command must correspond to the type of listener and authentication used to access the Kafka cluster.
-
-. Enable the Topic Operator by redeploying the Kafka cluster with the `topicOperator` property to recreate the `KafkaTopic` resources.
-+
-For example:
-+
-[source,shell,subs="+quotes,attributes"]
+.Kafka topic status
+[source,shell,subs="+quotes"]
 ----
-apiVersion: {KafkaApiVersion}
-kind: Kafka
-metadata:
-  name: my-cluster
-spec:
-  #...
-  entityOperator:
-    *topicOperator: {}* <1>
-    #...
+NAME         CLUSTER     PARTITIONS  REPLICATION FACTOR READY
+my-topic-1   my-cluster  10          3                  True
+my-topic-2   my-cluster  10          3                  True
+my-topic-3   my-cluster  10          3                  True
 ----
---
-<1> Here we show the default configuration, which has no additional properties.
-You specify the required configuration using the properties described in the link:{BookURLConfiguring}#type-EntityTopicOperatorSpec-reference[`EntityTopicOperatorSpec` schema reference^].
-
-. Verify the recovery by listing the `KafkaTopic` resources:
 +
-[source,shell,subs="+quotes,attributes"]
-----
-kubectl get KafkaTopic
-----
+Topic creation is successful when the `READY` output shows `True`.
```

Comment:
It doesn't really _create_ the topics as they existed all along. I would maybe just say something like

```
Topics are ready when the `READY` output shows `True`.
```

Or focus on the Kube resource

```
`KafkaTopic` custom resource creation is successful when the `READY` output shows `True`.
```